components/suggestion/at_mention_provider/at_mention_suggestion.tsx

```diff
@@ -113,7 +122,7 @@ export default class AtMentionSuggestion extends Suggestion {
                         <span className='suggestion-list__icon suggestion-list__icon--large'>
                             <i
                                 className='icon icon-account-multiple-outline'
-                                title={title}
+                                title={title as ReactNode as string}
```

Comment:
Hi @M-ZubairAhmed, The issue here is, this **i tag** is written inside a formattedMessage as a children, so this title can only be of type **React.ReactNodeArray**,
Castinig this directly to string gives **Conversion of type 'ReactNodeArray' to type 'string' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first** type mismatch error.

Reply:
ahh i see. How about we use the `formatMessage` with `injectIntl` then we dont have to deal with above pattern.
```
title={formatMessage(......
```

Reply:
@M-ZubairAhmed , we need to provide a children to this intl.formatMessage.
formatMessage does not accept a children.
![Screenshot from 2022-10-01 20-00-15](https://user-images.githubusercontent.com/89139362/193414320-a0d5b0ce-d415-4e13-a9db-0b1741917ea0.png)

Reply:
i mean we dont have to pass it via child of FormattedMessage
```suggestion
icon = (
                <span className='suggestion-list__icon suggestion-list__icon--large'>
                    <i
                        className='icon icon-account-multiple-outline'
                        title={formatMessage({id: 'generic_icons.member', defaultMessage: 'Member Icon'})}
                    />
                </span>
            );
```